test/e2e/error.test.js

```diff
@@ -0,0 +1,60 @@
+import Browser from '../utils/browser'
+import { loadFixture, getPort, Nuxt } from '../utils'
+
+let port
+const browser = new Browser()
+const url = route => 'http://localhost:' + port + route
+
+let nuxt = null
+let page = null
+
+describe('basic browser', () => {
+  beforeAll(async () => {
+    const config = await loadFixture('error')
+    nuxt = new Nuxt(config)
+    await nuxt.ready()
+
+    port = await getPort()
+    await nuxt.server.listen(port, 'localhost')
+
+    await browser.start({
+      // slowMo: 50,
+      // headless: false
+    })
+  })
+
+  test('Open /', async () => {
+    page = await browser.page(url('/'))
+
+    expect(await page.$text('h1')).toBe('Error Loop incoming page')
+  })
+
+  test('/squared doesnt loop due to error on error page', async () => {
+    await page.nuxt.navigate('/squared')
+
+    expect(await page.$text('h2')).toBe('An error occured while showing the error page')
+  })
+
+  test('/about loads normally', async () => {
+    await page.nuxt.navigate('/about')
+
+    expect(await page.$text('h1')).toBe('About')
+  })
+
+  test('/info prints empty page', async () => {
+    await page.nuxt.navigate('/info')
+
+    expect(await page.$text('#__layout')).toBe('')
```

Comment:
Is this correct behaviour? In dev mode it returns an error page, with build/start it doesnt

Reply:
Well it should displayed 'An error occured` error page I believe 🤔

Reply:
I checked with the dev branch and their the page is empty as well.

We could capture this error though as its being passed through errorCaptured. It just seems that by default in production there is no error handler so it just blanks out with an error in the console.

Reply:
loading-indicator will go to a bad state in case of fatal errors and SPA mode. Otherwise, that blank screen is somehow unexpected.